.github/actions/deploy-artifacts/action.yml

```diff
@@ -39,23 +49,35 @@ runs:
       run: rm -rf ${{ github.workspace }}/gh-docs/${DOCUMENTATION_PUBLISH_FOLDER}/*
       shell: bash
 
+    - name: Remove Previous App
+      run: rm -rf ${{ github.workspace }}/gh-docs/${APP_PUBLISH_FOLDER}/*
+      shell: bash
+
     - name: Fetch Documentation
       uses: actions/download-artifact@v3
       with:
         name: documentation
         path: ${{ github.workspace }}/gh-docs/${{env.DOCUMENTATION_PUBLISH_FOLDER}}
+    
+    - name: Fetch App
+      uses: actions/download-artifact@v3
+      with:
+        name: app
+        path: ${{ github.workspace }}/gh-docs/${{env.APP_PUBLISH_FOLDER}}
 
-    - name: Push Documentation
+    - name: Push Documentation and App
       run: |
         cd ${{ github.workspace }}/gh-docs
         git config --global user.name 'github-actions[bot]'
         git config --global user.email 'github-actions[bot]@users.noreply.github.com'
         git add -A ${DOCUMENTATION_PUBLISH_FOLDER}
         git commit -m "${DOCUMENTATION_PUBLISH_MESSAGE}"
+        git add -A ${APP_PUBLISH_FOLDER}
+        git commit -m "${APP_PUBLISH_MESSAGE}"
```

Comment:
Separate commits & messages seem like unnecessary extra work - is there any advantage I'm not seeing? 🤔
A single commit/push will simplify/reduce the git tree of the repo. It probably won't make much of a difference, but it might help reduce push/sync conflicts.

Reply:
No special reason behind doing separate commits. It was just a personal preference. 

In the end I had to change this code because [it's not possible to use deployment more than once in the same scope](https://github.com/lumada-design/hv-uikit-react/actions/runs/5176135648/jobs/9324752139). However, if I use it only once to deploy both the app and documentation, it's only possible to set one environment url when finishing the development status. Thus, I now use the matrix strategy from GitHub to be able to run this `deploy-artifacts` for both the app and documentation and have environment urls for both.